stdlib/universe/fill.go

```diff
@@ -255,14 +261,32 @@ func (t *fillTransformation) fillTable(w *table.StreamWriter, cr flux.ColReader,
 		return nil
 	}
 	vs := make([]array.Interface, len(w.Cols()))
+	var colArr array.Interface
```

Comment:
Can you move this into the for loop? I don't like having variable references outside of the scope that they are used in. I like to try and keep the variable initialization as close and into as narrow of a scope as where it is used to avoid dangling references.